string ssid
string ip
string mac
